pkgs/development/mobile/androidenv/generated/system-images-android.nix

```diff
@@ -281,6 +281,28 @@
       sha1 = "ce03c42d80c0fc6dc47f6455dbee7aa275d02780";
     };
   };
+  "29".default."x86" = {
+    name = "system-image-29-default-x86";
+    path = "system-images/android-29/default/x86";
+    revision = "29-default-x86";
+    displayName = "Intel x86 Atom System Image";
+    archives.all = fetchurl {
+      url = "https://dl.google.com/android/repository/sys-img/android/x86-29_r07-windows.zip";
+      sha1 = "e0e6504734f601c28350528d39a4b17b3e0f82dc";
+    };
+  };
+  "29".default."x86" = {
```

Comment:
This is the same attr as the one above. It looks like https://github.com/NixOS/nixpkgs/issues/95512 is still a problem. I guess we need to tweak generate.sh /cc @svanderburg